### PR TITLE
In Diff/Commit view display »Stage« Button consistently at the left

### LINKED ISSUE #9: Fix the stash/discard/copy buttons
This

capture d ecran 2016-12-20 10 04 00

makes me nervous.

I know I'll misclick once, and lose stuff. I know.

Propostions :
- Standardize on an ordering. Easy to do, but breaks muscle memory.
- Remove Copy, and make it go through the responder chain, as it should have. Harder, esp. w.r.t the state of the chain, but less UI clutter, and misclick chance gets `NULL`ed.

### PR DESCRIPTION
**Pull request by ssp**
This should fix #9.

Also display both action buttons for the selected
lines in the same colour (white) as they also have
the same colour for each hunk.

### CODE REVIEW

File: html/views/commit/commit.css [hunk 1]
```diff
@@ -94,8 +94,9 @@ table.diff {
     background-color: #7795c7;
 }
 
-#selected #stagelines {
+#selected .hunkbutton {
     background-color: #FFF;
+	box-shadow: 0px 0px 1px #cce5ff;
```
Comment (nanotech) [inline html/views/commit/commit.css]: Why add a box-shadow? It's fine, but it's quite subtle and the other buttons don't have a glow.

**Comment by ssp** [comment]
I added the drop-shadow because it can happen that the button is taller than a (single line) selection – which ends up looking weird when you have a white button on a white background. Like this there’s at least a hint of the button’s shape.

bildschirmfoto 2017-01-02 um 08 27 28

(An alternative solution would be using border instead of the drop-shadow but that would require reconsidering the padding for all the buttons.)

**Comment by nanotech** [comment]
Oh, that makes sense, thanks. For the color then, maybe something slightly darker?

Matching the lighter selection background (#b5d5fe):
screen shot 1
screen shot 2

Matching the darker selection background (#9abbea):
screen shot 3
screen shot 4

Although my color suggestions here could go in a different PR, since its rather tangential to moving the stage button.

**Comment by ssp** [comment]
I like the idea about the darker frame colour and also use a proper border now which should give a clearer line.

bildschirmfoto 2017-01-12 um 00 28 41

**Comment by nanotech** [comment]
Yeah, that looks good too. That's how [the border was originally][1] before I [removed it][2] when I made the buttons larger.

[1]: http://gitx.frim.nl/images/UserManual/commitview.png
[2]: https://github.com/gitx/gitx/commit/d9d6abcd67a4439517d9227fee6f32407796977b#diff-78156a5bce1a0be7d767603a2edafc7eL73

**Comment by ssp** [comment]
So is this one good to go then?

**Comment by nanotech** [comment]
LGTM 👍

**Pull request merged**